packages/unity-interface/dcl.ts

```diff
@@ -111,12 +111,12 @@ const unityInterface = {
     }
     gameInstance.SendMessage('SceneController', 'LoadParcelScenes', JSON.stringify(parcelsToLoad[0]))
   },
-  sendSceneMessage(parcelSceneId: string, method: string, payload: string) {
+  sendSceneMessage(parcelSceneId: string, method: string, payload: string, tag?: string) {
```

Comment:
In this case I think that `tag` should be optional
```suggestion
  sendSceneMessage(parcelSceneId: string, method: string, payload: string, tag: string) {
```

Reply:
isn't it?

Reply:
should NOT be optional perhaps?

Reply:
No